### PR TITLE
Rearrange methods of resource names utility in alphabetic order and add missing unit tests

### PR DESCRIPTION
**Pull request by PierrickVoulet**
Change-Id: I8018f8f66d34bb885064609f05f224230cdf167d

### CODE REVIEW

**Review by fiboknacky — COMMENTED** [review]
Could you highlight what new tests you added?

**Comment by PierrickVoulet** [comment]
> Could you highlight what new tests you added?

Sure. Added `testGetNameForTopicView` and `testGetNameForFeedPlaceholderView`.

File: tests/Google/Ads/GoogleAds/Util/V6/ResourceNamesTest.php [hunk 1]
```diff
@@ -2403,6 +2427,31 @@ public function testGetNameForTopicConstant()
...
+        $this->assertEquals($expectedResourceName, ResourceNames::forTopicView(
+            self::CUSTOMER_ID,
+            $adGroupId,
+            $criterionId
+        ));
+
+        $names = TopicViewServiceClient::parseName($expectedResourceName);
+        $this->assertEquals(self::CUSTOMER_ID, $names['customer_id']);
+        $this->assertEquals($adGroupId, $names['ad_group_id']);
+        $this->assertEquals($criterionId, $names['criterion_id']);
```
Comment (fiboknacky) [inline tests/Google/Ads/GoogleAds/Util/V6/ResourceNamesTest.php:2452]: So, the TopicViewServiceClient generates ad_group_id and criterion_id for us?

Comment (PierrickVoulet) [reply tests/Google/Ads/GoogleAds/Util/V6/ResourceNamesTest.php:2452]: I believe so, would not the assert fail if that is not the case? What makes you think that there is something wrong?

Comment (fiboknacky) [reply tests/Google/Ads/GoogleAds/Util/V6/ResourceNamesTest.php:2452]: The new scheme sometimes parse a name and generates tokens that are close to the resource name, e.g., 'customer_id' and 'topic_view'.
This case doesn't follow that scheme though.

**Review by fiboknacky — APPROVED** [review]

**Pull request merged**